### DISCUSSION TITLE
user-guide/plugins/translate

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#138] [文档评论区]
# user-guide/plugins/translate

Translate for Zotero 提示 深度适配 Zotero 阅读器的翻译插件。 项目主页：https://github.com/windingwind/zotero-pdf-translate 下载地址： GitHub Release Zotero 中文社区插件商店 Zotero PDF Translate 是由 @windingwind ...

https://zotero-chinese.github.io/user-guide/plugins/translate.html

**Comment by MiaoFiona** [comment]
我的zotero6之前总是自动闪退，我就卸载之后安装了7。现在没闪退现象了，但是translate for zotero的翻译界面不能在窗口右侧边显示了，请问是什么原因？

**Reply by northword** [reply]
尚未适配新版 Zotero 7 （beta 55及以上） 的侧边栏，耐心等待即可。

**Comment by fxvvv** [comment]
Zotero-6.0.36有适配的版本吗

**Reply by northword** [reply]
![image](https://github.com/zotero-chinese/wiki/assets/44738481/c369e623-1f63-4c67-83ff-0a9da0860428)

**Comment by zzz112225** [comment]
想问下为什么我的工具栏上面找不到附加组件这一栏了，这是什么原因呢？

**Reply by windingwind** [reply]
附加组件=插件

**Comment by Bai0213** [comment]
在编辑-首选项-翻译的设置里关闭了自动翻译标注后还在标注的ccomment里加自动翻译结果

**Reply by Bai0213** [reply]
已经解决了，原来是因为其它设备没有关这一设置，标注被同步过去后翻译了

**Comment by NNsAma** [comment]
那个翻译引擎效果比较好呀

**Comment by IcedBlackTea98** [comment]
划词翻译的时候，有些文档的英文会出现错误。比如：empirical会划取成eemmppiirriiccaall. 
我感觉是文档问题？

**Reply by windingwind** [reply]
文件问题

**Comment by lu990** [comment]
其他框里的摘要显示不全怎么办

**Comment by lxy7529** [comment]
请问为什么无法翻译标题，翻译摘要呢

**Reply by northword** [reply]
确认插件版本最新，确认翻译引擎可用，确认条目语言不是zh

**Comment by lsmyyds666** [comment]
你好，怎么选择目标语言，现在选中文本后不会翻译成中文

**Comment by dododemi** [comment]
你好，我的shift+p全文翻译之前一直正常，突然全文翻译时会显示request timed out after 60000 ms，请问怎么处理

**Comment by oaiyayayao945** [comment]
您好，请问微软翻译引擎的密钥如何获取呢？

**Reply by northword** [reply]
- https://github.com/windingwind/zotero-pdf-translate?tab=readme-ov-file#service

- https://github.com/windingwind/zotero-pdf-translate/issues/3#issuecomment-1064688597

**Comment by LEIXINRONG1995** [comment]
为什么我的翻译出现是这个问题？翻译引擎不可用，可能是密钥错误或可能请求太快

**Reply by northword** [reply]
根据报错信息里的文档排查。

**Comment by Doppa24** [comment]
请问现在是没有适配zotero 6的翻译插件了嘛？

**Reply by northword** [reply]
插件商店下载。

**Comment by HJHXD** [comment]
请问如何能实现翻译标题后自动代替原来的条目英文标题？

**Comment by OSTEW-1** [comment]
ZOTERO7 文献标题怎么变成中文？

**Comment by HJHXD** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by Pakwa-github** [comment]
zotero7 翻译窗口能否做成独立的小窗，可以拖动那种，和点其他地方也能持久存在

**Comment by HJHXD** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by laureline820** [comment]
请问划词翻译为什么会出现英文中间自动生成按顺序排列的数字的情况

**Comment by Mywidesky** [comment]
选中后的翻译框能否移动啊

**Comment by DejaVuDarren** [comment]
免费deepl，互惠互利，每人20w免费额度！！！
邀请码:
95Rv2vpTLE
邀请链接：
https://deepl-pro.com/#/translate?referral_code=95Rv2vpTLE

**Comment by DejaVuDarren** [comment]
免费deepl，互惠互利，每人20w免费额度！！！
邀请码:
O_rBwZG8Iw
邀请链接：
https://deepl-pro.com/#/translate?referral_code=O_rBwZG8Iw

**Comment by liquidweb9** [comment]
请问一下我安装插件之后：加入笔记出现在页面显示功能失效，但是禁用插件之后变正常，这个问题可以解决吗？

**Comment by dushuyyds** [comment]
尊敬的开发者你好，请求帮我解决译文不显示的问题。我的软件为zotero7、百度 api， 使用插件后翻译框内显示英文原文，并未显示中文译文。

**Comment by Junge-go** [comment]
单词发音怎么点了没反应？

**Comment by fxvvv** [comment]
mac系统上的zotero有中文翻译插件吗

**Comment by waiting1586** [comment]
安装了插件之后，点击翻译标题没有反应。而且我已经部署好了api，翻译pdf正文是没有问题的。请问这个问题怎么解决

**Comment by waiting1586** [comment]
点击标题翻译和摘要翻译没反应

**Comment by CharlieWangite** [comment]
为什么我用zotero7插件市场下载告诉我不适配呢

**Comment by lidongrunldr** [comment]
你好，插件导入的时候总显示版本不兼容，6和7都试过了。

**Comment by henry085706** [comment]
请问如何移动这个框，有时候翻译的时候要挡着图

**Comment by DarthYZK** [comment]
你好，我发现右侧信息栏的标题翻译无法同步中间的文献标题翻译，而且会有所有文献的标题翻译都变成同一个的问题。在更改了右侧信息栏的标题翻译之后，中间文献的标题翻译会同步更改，但是再在文献右键翻译标题，两边的标题翻译就都不会改变了。

**Reply by windingwind** [reply]
等待zotero更新

**Comment by blackr-abbit** [comment]
539779493
deeplx翻译为什么突然不能用了，两天了，想问一下解决方法

**Reply by blackr-abbit** [reply]
zotero版本7.0.10

**Comment by SaadSun** [comment]
升级到Zotero7之后右键菜单没有”切换到翻译后标题“这一功能了，设置中重新安装插件也未解决。请问是什么原因呢？

**Comment by zjn579379** [comment]
[请求错误]
此翻译引擎不可用，可能是密钥错误，也可能是请求过快。
可以尝试其他翻译引擎，或者来此查看相关回答：
https://zotero.yuque.com/staff-gkhviy/pdf-trans/age09f

请注意，这些错误与 Zotero 和本翻译插件无关，由该翻译服务引起： DeepLX

Error: HTTP POST https://www2.deepl.com/jsonrpc failed with status code 429

最近总是遇到这个问题，请问是怎么回事啊

**Comment by Balloon-Free** [comment]
双方均可以获取20w字符量，永久有效!
邀请码:
nGTzLPXlTU
邀请链接：
https://deepl-pro.com/#/translate?referral_code=nGTzLPXlTU

**Comment by HJHXD** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
